spec.bs

```diff
@@ -3050,45 +3056,65 @@ A <dfn>server auction response</dfn> is a [=struct=] with the following [=struct
+  :: A [=list=] of [=generated bid/interest group=] [=interest group/owner=] and
+     [=generated bid/interest group=] [=interest group/name=] pairs that bid
      in this auction.
   : <dfn>score</dfn>
-  :: {{double}} or Null. For component auctions, contains the desirability
-     for the winning bid in the component auction.
+  :: Null or {{double}}. Null when the server auction is not a component auction.
+     For component auctions, contains the [=leading bid info/top score=]
+     from the component auction executed on the trusted auction server.
   : <dfn>bid</dfn>
-  :: {{double}} or Null. For component auctions, contains the modified bid
-     for the winning bid in the component auction.
+  :: Null or {{double}}. Null when the server auction is not a component auction.
+     For component auctions, contains the
+     [=leading bid info/leading bid=]'s [=generated bid/modified bid=]'s
+     [=bid with currency/value=] if the [=generated bid/modified bid=] is not
+     null, otherwise the [=leading bid info/leading bid=]'s [=generated bid/bid=]'s
+     [=bid with currency/value=] from the component auction executed on the
+     trusted auction server.
   : <dfn>bid currency</dfn>
-  :: [=currency tag=] or Null. For component auctions, contains the currency of
-     the modified bid for the winning bid in the component auction.
+  :: Null or [=currency tag=]. Null when the server auction is not a component auction.
+     For component auctions, contains the
+     [=leading bid info/leading bid=]'s [=generated bid/modified bid=]'s
+     [=bid with currency/currency=] if the [=generated bid/modified bid=] is not
+     null, otherwise the [=leading bid info/leading bid=]'s [=generated bid/bid=]'s
+     [=bid with currency/currency=] from the component auction executed on the
+     trusted auction server.
   : <dfn>top level seller</dfn>
-  :: An [=origin=] or Null. Null when the server response is not feeding into an
-     on-device auction. Otherwise the value is the expected top-level seller for
-     that auction.
+  :: Null or an [=origin=]. Null when the server auction is not a component auction.
+     Otherwise the value is the expected top-level seller [=origin=] for
+     that auction. This should match the [=auction config/seller=] for the top-level [=auction config=].
   : <dfn>ad metadata</dfn>
-  :: A JSON [=string=] or Null. Null when the server response is not feeding into an
-     on-device auction. Otherwise the value contains the ad metadata
-     returned from scoring the winning bid in the component auction.
+  :: Null or a JSON [=string=]. Null when the server auction is not a component auction.
+     Otherwise the value contains the [=leading bid info/leading bid=]'s
+     [=generated bid/ad=] from the component auction executed on the
+     trusted auction server.
   : buyer reporting id
   :: TODO.
   : buyer and seller reporting id
   :: TODO.
   : error
   :: TODO.

```

Comment:
what is this used for

Reply:
I think it was intended to be used to send an error message back from the server. We aren't using it right now, so we can probably remove it from the spec.